global-script.lua

```diff
@@ -2975,7 +3110,20 @@ function setupPlayerArea(params)
         --Updates the number display
     end
 
-    Wait.time(countItems,1,-1)
+    initialized = true
+    obj.setVar("initialized", true)
```

Comment:
this is already being done on line 2795 right?

Reply:
Whoops.  Fixed.